CHANGELOG.md

```diff
@@ -29,6 +29,7 @@ This release has an [MSRV][] of 1.82.
 ### Changed
 
 * The `mul_alpha` method was renamed to `multiply_alpha`. ([#65][] by [@waywardmonkeys][])
+* Stray parenthesis in hex serialization of Rgba8 fixed ([#78][] by [@raphlinus][])
```

Comment:
For future reference, I'd probably put this in the `### Fixed` section

Reply:
Yeah, feel free to roll that into a future PR. I'm not sure why I didn't do that. Also I noticed after that the convention is a final period, which I missed.